test/e2e/network/netpol/kubemanager.go

```diff
@@ -112,11 +118,16 @@ func (k *kubeManager) probeConnectivity(nsFrom string, podFrom string, container
 	var cmd []string
 	switch protocol {
 	case v1.ProtocolSCTP:
-		cmd = []string{"/agnhost", "connect", fmt.Sprintf("%s:%d", addrTo, toPort), "--timeout=1s", "--protocol=sctp"}
+		cmd = []string{"/agnhost", "connect", fmt.Sprintf("%s:%d", addrTo, toPort), "--timeout=3s", "--protocol=sctp"}
```

Comment:
Curious, what's the story with SCTP on windows?  

I see that we're failing for UDP on windows, but we're only running TCP on windows.  Maybe add a comment to clarify the protocol/OS situation?

Reply:
I will add a comment